README.md

```diff
@@ -29,12 +29,12 @@ Hardware Required:
 
 See [Hardware Tested Section](https://github.com/viam-modules/lorawan?tab=readme-ov-file#hardware-tested) for hardware we have used this module with successfully.
 
-## Configure the `viam:sensor:sx1302-gateway`
+## Setup the `viam:sensor:sx1302-gateway`
 
 Navigate to the **CONFIGURE** tab of your machine in the [Viam app](https://app.viam.com/) and click the **+** button.
 [Add sx1302-gateway to your machine](https://docs.viam.com/operate/get-started/supported-hardware/#configure-hardware-on-your-machine).
 
```

Comment:
moved this so it doesn't show up in markdown, when the config is already configured